vscode/src/projectSystem.ts

```
@@ -145,7 +162,12 @@ async function readFileUri(
       };
     });
   } catch (_err) {
-    // `readFile` returns `err` if the file didn't exist.
+    // `readFile` should throw the below if the file is not found
+    if (
+      !(_err instanceof vscode.FileSystemError && _err.code === "FileNotFound")
```

Drive by improvement to log an error if unexpected, rather than swallow all.